test/test_suite.py

```diff
@@ -3095,7 +3095,7 @@ def test_all_default_options(self):
         retcode, o, e = run_command(cmd, env_to_inject=env_to_inject)
 
         self.assertEqual(retcode, 0)
-        self.assertEqual(len(o), 34)
+        self.assertEqual(len(o), 35)
```

Comment:
(see discussion above) this is why I had to increase this by 1

Reply:
see my comment above.